examples/vitejs-with-typescript/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```

Comment:
@miha53cevic is it an unused extra line?
`<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />`

Can you point to any file of this PR that uses this font?

Reply:
It's not used anywhere, I always thought that you had to add it after installing the @mui/icons-material package. My mistake.